src/main/java/converter/StringToUserConverter.java

```diff
@@ -7,8 +7,9 @@ public class StringToUserConverter {
     public User convert(String line) {
 
         if (line == null || line.isBlank()) {
-            throw new IllegalArgumentException("string = null");
+            throw new IllegalArgumentException("String = null");
```

Comment:
```suggestion
            throw new IllegalArgumentException("line = null");
```
as your check line in `if`, not string

Reply:
still actual